docs/snapshotting/snapshot-support.md

```diff
@@ -0,0 +1,277 @@
+# Firecracker Snapshotting
+
+## What is microVM snapshotting?
+
+MicroVM snapshotting is a mechanism through which a running microVM, including
+the guest environment, can be serialized and saved to an external medium in the
+form of a `snapshot`. This snapshot can be later used to restore a microVM
+with its guest workload at that particular point in time.
+
+## Snapshotting in Firecracker
+
+Firecracker snapshotting allows the creation of a microVM snapshot. This
```

Comment:
`Firecracker snapshotting allows the creation of a microVM snapshot.` is a bit redundant, I think it can be removed.